samples/apps/copilot-chat-app/webapi/Storage/ChatMessageRepository.cs

```diff
@@ -21,9 +21,9 @@ public ChatMessageRepository(IStorageContext<ChatMessage> storageContext)
     /// <summary>
     /// Finds chat messages by chat id.
     /// </summary>
-    public Task<IEnumerable<ChatMessage>> FindByChatIdAsync(string chatId)
+    public async Task<IEnumerable<ChatMessage>> FindByChatIdAsync(string chatId)
     {
-        return Task.FromResult(base.StorageContext.QueryableEntities.Where(e => e.ChatId == chatId).AsEnumerable());
+        return await base.StorageContext.QueryEntitiesAsync(e => e.ChatId == chatId);
```

Comment:
Seems like we should be able to drop both the async and the await and just return from QueryEntitiesAsync?

Reply:
same for the other files in this PR